kubemarine/procedures/check_paas.py

```diff
@@ -1565,6 +1565,18 @@ def kubernetes_admission_status(cluster: KubernetesCluster) -> None:
         cluster.log.debug(kube_admission_status)
         tc.success(results='enabled')
 
+def verify_kubernetes_version(cluster: KubernetesCluster) -> None:
```

Comment:
Need to update https://github.com/Netcracker/KubeMarine/blob/main/documentation/Kubecheck.md#paas-procedure respectively

Reply:
done